src/components/expression/expression-renderer.js

```diff
@@ -389,10 +449,11 @@ genotet.ExpressionRenderer.prototype.drawMatrixGeneLabels_ = function() {
   var heatmapData = this.data.matrix;
   var geneLabelsData = heatmapData.geneNames;
   var cellHeight = this.heatmapHeight_ / geneLabelsData.length;
-  var transformTop = this.profileHeight_ + this.TEXT_HEIGHT_ / 2 +
+  var transformTop = this.profileHeight_ + this.TEXT_HEIGHT / 3 +
```

Comment:
If you really need to divide by 3, then probably you want to create another constant called TEXT_HEIGHT_OVER_THREE and then declare it in the prototype. In the comments of this variable you might explain why you are using this "dividing by 3" value.
```js
/**
 * A third of text height. Using a-third of the text height is really important because ...
 * @const {number}
 */
genotet.ExpressionRenderer.TEXT_HEIGHT_OVER_THREE = ...
```